content/en/docs/tasks/administer-cluster/kubelet-config-file.md

```diff
@@ -38,7 +38,7 @@ evictionHard:
 ```
 
 In the example, the Kubelet is configured to serve on IP address 192.168.0.8 and port 20250, pull images in parallel,
-and evict Pods when available memory drops below 200Mi.
+and evict Pods when available memory drops below 200Mi. Since only one of the four evictionHard thresholds is configured, other evictionHard thresholds are reset to 0 from their built-in defaults.
```

Comment:
Please wrap the long lines appropriately as we did for other paragraphs.

Reply:
thanks @tengqm  - Updated the PR.